Tweeter/DAL/TweeterRepository.cs

```diff
@@ -2,19 +2,35 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Web;
+using Microsoft.AspNet.Identity;
+using Tweeter.Models;
 
 namespace Tweeter.DAL
 {
     public class TweeterRepository
     {
+        
+
         public TweeterContext Context { get; set; }
+        public UserManager<ApplicationUser> UserManagerContext { get; set; }
         public TweeterRepository(TweeterContext _context)
         {
             Context = _context;
         }
 
-        public TweeterRepository() {}
+        public TweeterRepository()
+        {
```

Comment:
You need to make a new instance of of the context here -- otherwise your tests will pass but when you actually run your code you will get a null reference exception.